packages/internal/src/generate/templates/all-currentUser.d.ts.template

```diff
@@ -6,9 +6,20 @@ import { getCurrentUser } from '../../../api/src/lib/auth'
 type ThenArg<T> = T extends PromiseLike<infer U> ? U : T
 export type InferredCurrentUser = ThenArg<ReturnType<typeof getCurrentUser>>
 
+type UndefinedRoles = {
+  /**
+   * ⚠️ Heads-Up: This is undefined unless roles key is returned from {@link getCurrentUser()}
+   *
+   * You may look into to https://redwoodjs.com/docs/tutorial/chapter7/rbac for a tutorial on
+   * how to add fully-fledged RBAC (Role-based Access Control) to your database model.
+   */
+  roles?
+}
+
+type Overwrite<T, U> = Pick<T, Exclude<keyof T, keyof U>> & U;
```

Comment:
Just adding a blank line here to make the code more readable (and to keep with the style of the rest of the file)
```suggestion
type Overwrite<T, U> = Pick<T, Exclude<keyof T, keyof U>> & U;

```

Reply:
Actually i tried to follow the pattern above that line.
```
type ThenArg<T> = T extends PromiseLike<infer U> ? U : T
export type InferredCurrentUser = ThenArg<ReturnType<typeof getCurrentUser>>
```
which in my eyes makes more sense, as they are a couple that goes together. If `Overwrite` was an export, i would have had put it separately.

But i have no strong opinion on this here, hence: thy wish be to me command :bow:  – fixed via 003fd224e1d1da6536819e691e37af0928996890